<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewPanel</class>
 <widget class="QWidget" name="PreviewPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>597</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: black;
}

QFrame{
	background-color: rgb(45,45,45);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QFrame" name="TopFrame">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>355</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="PreviewLabel">
         <property name="styleSheet">
          <string notr="true">QLabel{
	background-color: black;
	
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="DetailFrame">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>138</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: white;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="authorLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>19</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>8</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>15</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="titleLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Black</family>
              <pointsize>13</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>15</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="descriptionLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>78</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>8</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="indent">
             <number>15</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="BottomFrame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>220</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
	color: white;
}
QLabel{
	color: white;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="LoadOrderLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Variable Small</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Load Order</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
         <property name="indent">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="ListWidget">
         <property name="font">
          <font>
           <family>Segoe UI Variable Display Semib</family>
           <pointsize>9</pointsize>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget{
	padding-top: 3px;
	padding-left: 6px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="autoScrollMargin">
          <number>16</number>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragOnly</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::TargetMoveAction</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideRight</enum>
         </property>
         <property name="movement">
          <enum>QListView::Snap</enum>
         </property>
         <property name="isWrapping" stdset="0">
          <bool>false</bool>
         </property>
         <property name="layoutMode">
          <enum>QListView::SinglePass</enum>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="viewMode">
          <enum>QListView::ListMode</enum>
         </property>
         <property name="modelColumn">
          <number>0</number>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
         <property name="selectionRectVisible">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
